backend/api/routes/tournament_player/router.go

```diff
@@ -19,6 +19,8 @@ func RegisterEndpoints(r *mux.Router) {
 	r.HandleFunc("", GetTournamentPlayersFromAuthHandler).Methods(http.MethodGet)
 	r.HandleFunc("/tournament", GetTournamentPlayer).Methods(http.MethodGet)
 	r.HandleFunc("", CreateTournamentPlayerHandler).Methods(http.MethodPost)
+	r.HandleFunc("/add/coins", AddCoinsToTournamentPlayerHandler).Methods(http.MethodPost)
+	r.HandleFunc("/add/points", AddPointsToTournamentPlayerHandler).Methods(http.MethodPost)
```

Comment:
I would push for trying to keep our urls restfuls

I think this could look like 
POST to `/coins`
POST to `/points`